common/artifacts/901_wc_artifacts_weapon_onehanded.txt

```diff
@@ -53,4 +53,30 @@ hammer_of_the_twilight = {
 
 	picture = "GFX_hammer_of_the_twilight"
 	slot = weapon
-}
\ No newline at end of file
+}
+ironfoe = {
+	quality = 4
+	flags = { combat_weapon hammer }
+
+	active = {
+		is_adult = yes
+		shared_physical_weapon_trigger = yes
```

Comment:
`is_adult = yes` is included in `shared_physical_weapon_trigger`.
```suggestion
		shared_physical_weapon_trigger = yes
```

Reply:
Alright, done.